src/dialogs/more-info/controls/more-info-cover.ts

```diff
@@ -93,12 +93,15 @@ class MoreInfoCover extends LitElement {
       supportsFeature(this.stateObj, CoverEntityFeature.CLOSE_TILT) ||
       supportsFeature(this.stateObj, CoverEntityFeature.STOP_TILT);
 
-    const supportsOpenCloseWithoutStop =
+    const supportsOpenCloseOnly =
       supportsFeature(this.stateObj, CoverEntityFeature.OPEN) &&
       supportsFeature(this.stateObj, CoverEntityFeature.CLOSE) &&
       !supportsFeature(this.stateObj, CoverEntityFeature.STOP) &&
       !supportsFeature(this.stateObj, CoverEntityFeature.OPEN_TILT) &&
-      !supportsFeature(this.stateObj, CoverEntityFeature.CLOSE_TILT);
+      !supportsFeature(this.stateObj, CoverEntityFeature.CLOSE_TILT) &&
+      !supportsFeature(this.stateObj, CoverEntityFeature.STOP_TILT) &&
```

Comment:
```suggestion
      !supportsTilt &&
```

`!supportsFeature(this.stateObj, CoverEntityFeature.OPEN_TILT) &&` needs to be removed.